target/linux/ath79/dts/qca9531_8dev_lima.dts

```diff
@@ -0,0 +1,123 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+        compatible = "8dev,lima", "qca,qca9531";
+        model = "8devices Lima";
+
+        aliases {
+                spiflash = &spiflash;
+        };
+
+        chosen {
+		bootargs="console=ttyS0,115200n81";
+        };
+
+	keys {
```

Comment:
Please check your indent. DTS should be tabs-only.

Reply:
Fixed.

Reply:
Not fixed in a lot of places in this file.

Reply:
Fixed.

Reply:
No.

Reply:
Ah, now.